src/middlewares/checkArticleOwnership.js

```diff
@@ -0,0 +1,47 @@
+import models from '../api/models';
+import statusCode from '../helpers/constants/status.codes';
+
+const { Article } = models;
+/**
+ * containing all user's model controllers (signup, login)
+ *
+ * @export
+ * @class CheckUserOwnership
+ */
+class CheckUserOwnership {
+  /**
+   * this is a middleware which checks if the user is registered in to our database.
+   *
+   * @author Frank Mutabazi, Prémices
+   * @static
+   * @param {object} req the request
+   * @param {object} res the response to be sent
+   * @param { object } next the next route controller to be called
+   * @memberof CheckUserOwnership
+   * @returns {Object} res
+   */
+  static async checkOwner(req, res, next) {
+    const {
+      user: { id }
+    } = req.user;
+    const { slug } = req.params;
+
+    const response = await Article.findOne({
```

Comment:
same here, why don't you handle both success and failure operations?

Reply:
@Ramadhan0 please attend to this feedback